IotTemplates/WindowsIotCoreTemplates/ProjectTemplates/WiringAppCpp/Package-native.appxmanifest

```diff
@@ -54,5 +54,10 @@
     <Capability Name="internetClient" />
     <iot:Capability Name="lowLevelDevices" />
     <DeviceCapability Name="109b86ad-f53d-4b76-aa5f-821e2ddf2141"/>
+    <DeviceCapability Name="serialcommunication">
```

Comment:
Hardware Serial, especially Serial.print(), which is popular in Arduino, requires the capability. So, adding it by default to the template will reduce the chances someone hits the missing declaration exception when the function is used.